.github/actions/update-with-latest-versions/entrypoint.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+set -eu
+
+BRANCH=bump-documentation-to-latest-versions
+LATEST_IPFS_TAG=$INPUT_LATEST_IPFS_TAG
+
+echo "The latest IPFS tag is ${LATEST_IPFS_TAG}"
+
+ROOT=`pwd`
+git checkout -b ${BRANCH}
+API_FILE=`pwd`/docs/reference/http/api.md
+
+
+# Update http api docs and cli docs
+
+cd tools/http-api-docs
+
+# extract go-ipfs release tag used in http-api-docs from go.mod in this repo
+CURRENT_IPFS_TAG=`grep 'github.com/ipfs/go-ipfs ' ./go.mod | awk '{print $2}'`
+echo "The currently used go-ipfs tag in http-api-docs is ${CURRENT_IPFS_TAG}"
+
+# make the upgrade, if newer go-ipfs tags exist
+if [ "$CURRENT_IPFS_TAG" = "$LATEST_IPFS_TAG" ]; then
+    echo "http-api-docs already uses the latest go-ipfs tag."
+else
+     # update http-api-docs
+     sed "s/^\s*github.com\/ipfs\/go-ipfs\s\+$CURRENT_IPFS_TAG\s*$/	github.com\/ipfs\/go-ipfs $LATEST_IPFS_TAG/" go.mod > go.mod2
+     mv go.mod2 go.mod
+     go mod tidy
+     make
+     http-api-docs > $API_FILE
+
+     # update cli docs
+     cd $ROOT # go back to root of ipfs-docs repo
+     git clone https://github.com/ipfs/go-ipfs.git
+     cd go-ipfs
+     git fetch --all --tags
+     git checkout tags/$LATEST_IPFS_TAG
+     go install ./cmd/ipfs
+     cd $ROOT/docs/reference
+     ./generate-cli-docs.sh
+fi
```

Comment:
Note: github doesn't detect the move so it looks like I created the whole file. This PR actually mainly move the update version code lower and calls it three times.

- We also push to a unique branch on file changes, instead of something like `new-documentation-${NEW_TAG}` because documentation updates are not only tied to go-ipfs versions anymore.

- I was thinking about extracting the version replace script to its own action, but I'm under the impression that would make the code more complex (spread it around 3 actions instead of a single script file).